### ISSUE TITLE
Pipe character in strings such as page title

### ISSUE THREAD

**Issue by niftylettuce** [mashpie/i18n-node#274]

**Comment by niftylettuce** [comment]
We can't have this get translated if it contains a pipe character.

`Foo Bar | Baz Beep` becomes `Foo Bar`

Also this pipe character is not documented anywhere in the Readme.

I found it here: <https://github.com/mashpie/i18n-node/blob/master/i18n.js#L531-L534>

**Comment by mashpie** [comment]
You may also find it here: https://github.com/mashpie/i18n-node#ranged-interval-support

For now my advice is to use `&#124;` instead of `|`.

**Comment by niftylettuce** [comment]
Aw, okay ☹️

**Comment by mashpie** [comment]
I'd still like to check back with a better test on ranges, though

**Comment by adrianotadao** [comment]
@niftylettuce Is there some news about it?

**Comment by niftylettuce** [comment]
I wrote something custom for this I believe in CrocodileJS.  I'll try to
remember to follow up here

**Comment by niftylettuce** [comment]
Ah, I know what I did @adrianotadao, take a look here if you're interested in my workaround...

> **tldr;** I basically escape and then unescape the HTML entity `|` to `&#124;`

<https://github.com/crocodilejs/crocodile-node-mvc-framework/blob/development/src/helpers/meta.js#L58-L67>

... and here ...

<https://github.com/crocodilejs/crocodile-node-mvc-framework/blob/development/src/config/meta.js#L20-L25>

... and in my translation helper with Google Translate here ....

<https://github.com/crocodilejs/crocodile-node-mvc-framework/blob/development/src/jobs/index.js#L267-L270>

**Comment by adrianotadao** [comment]
@niftylettuce hummm ... thank you a lot for your time. I will change something here 👏 😄
